### PR TITLE
Invoice text changes

### PR DESCRIPTION
**Pull request by nickpell**
See http://manage.dimagi.com/default.asp?226680 for changes

@esoergel

### CODE REVIEW

File: corehq/apps/accounting/invoice_pdf.py [hunk 1]
```diff
@@ -412,10 +414,26 @@ def draw_footer(self):
...
+            "<font color='blue'>%(invoicing_contact_email)s</font> "
+            "so that we can match your payment to the correct invoice.  Please include: "
+            "Invoice #, Project Space, and payment date in the email. <br />"
+        ) % {
+            'invoicing_contact_email': settings.INVOICING_CONTACT_EMAIL,
+            'payment_page': absolute_reverse(DomainBillingStatementsView.urlname, args=[self.project_name]),
+        }
+        payment_info = Paragraph('\n'.join([
+            payment_description,
+        ]), ParagraphStyle(''))
```
Comment (esoergel) [inline corehq/apps/accounting/invoice_pdf.py]: It looks like this is `'\n'.join`ing a list with a single element, what's that about?

Comment (nickpell) [inline corehq/apps/accounting/invoice_pdf.py]: this is obsoleted; removed

**Comment by esoergel** [comment]
:+1: on :construction:

**Comment by nickpell** [comment]
✅

**Pull request merged**